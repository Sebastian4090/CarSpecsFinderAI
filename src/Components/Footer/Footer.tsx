
const Footer = () => {
    return (
        <div className="w-full bottom-0 h-20 bg-indigo-800 flex justify-center items-center shadow-2xl">
                <h1 className="text-white text-center">FOOTER</h1>
        </div>
    )
}

export default Footer;