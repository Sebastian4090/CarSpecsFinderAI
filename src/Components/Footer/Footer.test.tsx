import Footer from "./Footer";

describe("<Footer /> Component", () => {
  it("renders correctly", () => {
    expect(<Footer />).toMatchSnapshot();
  });
});
