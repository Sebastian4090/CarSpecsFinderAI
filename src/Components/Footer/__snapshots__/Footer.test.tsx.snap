// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Footer /> Component > renders correctly 1`] = `<Footer />`;
